CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # Changelog
+## 2.0.1
+### Added
+
+*no additions*
+
+### Changed
+
+* altered the sample values for author preferred names in references and datasets
```

Comment:
Not a good description. Perhaps:

> author `preferred` field changed from "surname firstname" to "firstname surname" in references and datasets

Reply:
looking at the article changes I guess the "One Person" and "Person One" examples aren't that clear. If this is what is happening:

>\-                         "preferred": "C Darwin",
>\+                       "preferred": "Darwin C",

Then the description would be:

> author preferred field changed from "firstname surname" to "surname firstname" in references and datasets

Anyway, your decision.